nano/node/transport/tcp.hpp

```diff
@@ -68,6 +68,16 @@ namespace transport
 			return nano::transport::transport_type::tcp;
 		}
 
+		virtual bool full () override
+		{
+			bool result = true;
+			if (auto socket_l = socket.lock ())
+			{
+				result = socket_l->max ();
```

Comment:
This function is called full() but it calls a socket functions max().
Confusingly, there is a socket called full() as well which the channel function full() does not call.
I think this should be at least called max() to match the socket max.
Although, I think it would be better to call both max function something else, half_full() maybe?

I wonder if it would be better to have full(), half_full() and quarter_full() consistent across tcp sockets and tcp channels.